Commands/gemini3.py

```
@@ -126,33 +237,38 @@ def reply_with_grounded_gemini(self, message):
 
-    try:
         clean_result = clean_str(result, ['`', '*'])
         send_chunks(self.send_privmsg, cmd.channel, clean_result)
-        if duck_urls:
-            self.send_privmsg(cmd.channel, f"📝 Source(s): {' | '.join(duck_urls)}")
+        if valid_urls:
+            self.send_privmsg(cmd.channel, f"📝 Source(s): {' | '.join(valid_urls)}")
+
     except Exception as e:
         print(f"[Error] {e}")
         self.send_privmsg(cmd.channel, f"Failed to send a response. Please try again later")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)